csrc/ops/utils.h

```diff
@@ -51,11 +51,22 @@ std::vector<IterDomain*> mapMatmulOpIterDomains(
     MatmulRole input_role,
     size_t out_size);
 
+// For LinearOp, the output is the same as the first input (A[*, in_features])for all but the last dimension.
+// If the second input is 2D (B[out_features, in_features]), the last dimension of output is out_features.
+// If bias is 1D (bias[out_features]) it maps to the last dimension of the output.
+std::vector<IterDomain*> mapLinearOpIterDomains(
```

Comment:
Please document the `out_size` option since it is not obvious at first glance why it is needed.